src/i18n/dk.js

```diff
@@ -0,0 +1,43 @@
+export default {
+    header: {
+        slogan: 'Fremtidens webudvikling'
+    },
+    callToAction: {
+        start: 'Kom igang',
+        join: 'Join Chat'
+    },
+    nav: {
+        code: 'Kode',
+        cloud: 'Cloud',
+        docs: 'Dokumenter'
+    },
+    bio: {
+        first: 'Ikke-blokerende, event-dreven arkitektur bygget oven på Apple\'s <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO</a> leverer Vapor <a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">høj ydeevne</a>.',
+        second: 'Skrevet i <a target="_blank" href="https://swift.org">Swift</a>, det kraftfulde programmeringssprog, der samtidig er nemt at lære.',
+        third: 'Ekspressivt, <a target="_blank" href="https://developer.apple.com/videos/play/wwdc2015/408/">protokol-orienteret design,</a> med fokus på typesikkerhed og vedligeholdelse.'
+    },
+    contribute: {
+        title: 'Bidrag!',
+        options: {
+            code: {
+                description: 'Indsend forslag til nye funktioner, reporter fejl, og merge kode på Github!'
```

Comment:
Maybe this should be something like this: `Deltag i udviklingen, foreslå nye features eller reporter fejl på Github!`